src/template.proto

```diff
@@ -12,6 +12,8 @@ message AnalyzeTemplate {
     string createTime = 3;
     // Template modification date
     string modifiedTime = 4;
+    // Field type language code
+    string languageCode = 2;
```

Comment:
Two questions:
1. Shouldn't the value be 5?
2. Is this the template that's used both for configuring the analyzer and for a specific request?